art_craft/demo/product_template.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
```

Comment:
Did you test this? Because this file is not in the manifest, so never converted into the database...